### ISSUE TITLE
Support for SVG Filter Effects

### ISSUE THREAD

**Issue by dusipavan** [BigBadaboom/androidsvg#74]
May I know if there is any plan  to support SVG Filers?

I could see below comment in your Project description.
(https://code.google.com/p/androidsvg/wiki/ReleaseNotes)

Thanks

**Comment by BigBadaboom** [comment]
Yes.  I have done a lot of work on it, but it is not near to being ready for release yet.

**Comment by dusipavan** [comment]
Thanks for your quick reply.
I would like to use your parser for some basic SVG filters. 
Is it possible to share your current filter code or idea (which might help me to add filter feature in current code ).

**Comment by Emasoft** [comment]
I'll love to see those supported too. At least blur and drop-shadow, please.. I can't do anything without those..

**Comment by amanzan** [comment]
Shadow support would be fantastic :D
